Homarus/README.md

```diff
@@ -1,9 +1,29 @@
 # Homarus
 
-### Apache Config
-```
-# managed by Ansible
+## Introduction
+
+[FFmpeg](https://www.ffmpeg.org/) as a microservice.
+
+## Installation
+- Install `ffmpeg`.  On Ubuntu, this can be done with `sudo apt-get install ffmpeg`. 
+- Copy `/var/www/html/Crayfish/Homarus/cfg/config.default.yml` to `/var/www/html/Crayfish/Homarus/cfg/config.yml`
```

Comment:
Shuffle these two copy commands under cloning the repository so the files will exist.